commands/remove-container.ts

```diff
@@ -0,0 +1,55 @@
+import vscode = require('vscode');
+import { docker } from './utils/docker-endpoint';
+import { reporter } from '../telemetry/telemetry';
+import { DockerNode } from '../explorer/dockerExplorer';
+import { dockerExplorerProvider } from '../dockerExtension';
+import { ContainerItem, quickPickContainer } from './utils/quick-pick-container';
+
+const teleCmdId: string = 'vscode-docker.container.remove';
+
+export async function removeContainer(context?: DockerNode) {
+
+    let containersToRemove: Docker.ContainerDesc[];
+
+    if (context && context.containerDesc) {
+        containersToRemove = [context.containerDesc];
+    } else {
+        const opts = {
+            "filters": {
+                "status": ["created", "restarting", "running", "paused", "exited", "dead"]
```

Comment:
I used the options/filter to give us some flexibility with the explorer contribution. we may want to limit what we show in the tree in the future.